catapult-sdk/src/plugins/lockSecret.js

```diff
@@ -35,23 +35,25 @@ const lockSecretPlugin = {
 			lock: { type: ModelType.object, schemaName: 'secretLockInfo.lock' }
 		});
 		builder.addSchema('secretLockInfo.lock', {
-			account: ModelType.binary,
-			accountAddress: ModelType.binary,
+			senderPublicKey: ModelType.binary,
+			senderAddress: ModelType.binary,
 			mosaicId: ModelType.uint64,
 			height: ModelType.uint64,
 			secret: ModelType.binary,
-			recipient: ModelType.binary
+			recipientAddress: ModelType.binary
 		});
 
 		builder.addTransactionSupport(EntityType.secretLock, {
 			mosaicId: ModelType.uint64,
 			duration: ModelType.uint64,
+			amount: ModelType.uint64,
+			hashAlgorithm: ModelType.uint8,
```

Comment:
`hashAlgorithm` should not be here because it is a basic type (uint8, remove line)